app/controllers/datasource/show.js

```diff
@@ -337,6 +389,30 @@ Ember.Controller.extend(
         }.bind(this)
       );
 
-    }.property('model.dimensions.[]', 'runCount', 'interval', 'metric')
+    }.property('model.dimensions.[]', 'runCount', 'interval', 'metric'),
+
+    typeAheadList: function () 
```

Comment:
yeah...I changed it. However, I did that to remain consistent with how other computed properties are updated such as topKData. But now I think it makes more sense to directly call this method as it only depends on search query.